src/Services/Analytics.ts

```diff
@@ -26,13 +26,25 @@ class AnalyticsService {
     this.ga = new GA(options.ga);
```

Comment:
Add `transfer_initialized` event and use it.

Reply:
fixed to `transfer_initializing` to reflect current application  logic